api/src/test/java/org/openmrs/module/appointments/service/SpecialityServiceTest.java

```diff
@@ -58,4 +65,12 @@ public void shouldNotGetAllSpecialitiesIfUserHasNoPrivilege() throws Exception {
         Context.authenticate(noPrivilegeUser, password);
         assertNotNull(specialityService.getAllSpecialities());
     }
+    
+    @Test
+    public void shouldBeAbleToSaveSpecialityIfUserHasManageServicesPrivilege() throws Exception {
```

Comment:
We should also try to save when user does not have the privilege (_Manage Appt. Specialities_) and ensure it fails.

Reply:
Done